### ISSUE TITLE
Using easyForm in route

### ISSUE THREAD

**Issue by antoniusostermann** [sir-dunxalot/ember-easy-form-extensions#40]
(Ember: 1.13.3, EmberData: 1.13.5, easyFormExtensions: 1.0.2)

As you may know, using controllers is deprecated in ember. Because of this I never create any controller, doing everything in routes or components (like stated in http://thetechcofounder.com/getting-started-with-ember-js-using-ember-cli/). If I use this extension in my route like this:

``` javascript
import Ember from 'ember';
import FormMixin from 'ember-easy-form-extensions/mixins/controllers/form';

export default Ember.Route.extend(FormMixin, {
  model: function() {
    return this.store.find('post');
  },

  actions: {

    cancel: function() {
    },


    save: function() {
    },
  }
});
```

with this template:

``` hbs
{{#form-wrapper}}
  {{#form-controls legend='Write a new post'}}

    {{!--model.title--}}
    {{input-group property='title'}}

    {{!--model.description--}}
    {{input-group property='body' type='textarea'}}

  {{/form-controls}}

  {{!--Submit and cancel buttons--}}
  {{form-submission}}
{{/form-wrapper}}
```

I currently get 

> TypeError: formController is null

linking to 

``` javascript
   addBindingForErrors: on('didInitAttrs', function () {
      var property = this.get('property');

      Ember['default'].assert('You must set a property attribute on the {{error-field}} component', property);

      var formController = this.get('formController');
      var validations = formController.get('validations');
```

Could you maybe give an example on how to use this with routes (also to make it compatible with future ember versions)?

**Comment by Truffula** [comment]
AFAIK you need to keep using controllers until routable components land. `Ember.Route::renderComponents` won't be in the API for one or two more versions.

**Comment by sir-dunxalot** [comment]
The recent refactor was written with this exact use case in mind so it's likely that there is some property reference amiss rather than an architectural issue. My guess is that it's related to [this code](https://github.com/sir-dunxalot/ember-easy-form-extensions/blob/master/addon/components/form-wrapper.js#L28-38).

I'll need to make a test locally before I can confirm that though. We'll also need to increase the acceptance test coverage for this 'component usage' because this library is lacking that (in fact more acceptance tests need to be added in general).

**Comment by sir-dunxalot** [comment]
Perhaps we could also have a mixin at the path `ember-easy-form-extensions/mixins/components/form.js` to easy future upgrading - this would just export the controller mixin.

**Comment by sir-dunxalot** [comment]
Fixed with #48 (released as `2.0.0`).
